### PR TITLE
Remove duplicated IPAM code for NSE applications

### PR DESCRIPTION
**Pull request by bszirtes**
NSM issue link: https://github.com/networkservicemesh/sdk/issues/1409

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
-// Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
+// Copyright (c) 2021-2024 Doc.ai and/or its affiliates.
```
Comment (denis-tingaikin) [inline main.go:1]:
```suggestion
// Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
```

File: main.go [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 //
-// Copyright (c) 2023 Cisco and/or its affiliates.
+// Copyright (c) 2023-2024 Cisco and/or its affiliates.
```
Comment (denis-tingaikin) [inline main.go:3]:
```suggestion
// Copyright (c) 2023 Cisco and/or its affiliates.
```

**Review by denis-tingaikin — APPROVED** [review]
Let's not modify the copyright holders of other companies ;)  

In general, LGTM.

File: main.go (hunk 2 above)
Comment (bszirtes) [reply main.go]: Thank you, I updated it.

**Review by denis-tingaikin — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**